submissions/benjaminKamena/database/init.sql

```diff
@@ -0,0 +1,101 @@
+--
+-- Database: `ecommerce_store`
+--
+
+-- --------------------------------------------------------
+--
+-- Table structure for table `user_admins`
+--
+CREATE TABLE `user_admins` (
+  `admin_id` int(11) NOT NULL AUTO_INCREMENT,
+  `admin_name` varchar(100) NOT NULL,
+  `email_address` varchar(100) NOT NULL,
+  `admin_password` varchar(200) NOT NULL,
+  `phone_number` varchar(20) NOT NULL,
+  PRIMARY KEY (`admin_id`)
+);
+
+-- --------------------------------------------------------
+--
+-- Table structure for table `categories`
+--
+CREATE TABLE `categories` (
+  `category_id` int(11) NOT NULL AUTO_INCREMENT,
+  `category_name` varchar(100) NOT NULL,
+  PRIMARY KEY (`category_id`)
+);
+
+-- --------------------------------------------------------
+--
+-- Table structure for table `products`
+--
+CREATE TABLE `products` (
+  `product_id` int(11) NOT NULL AUTO_INCREMENT,
+  `product_name` varchar(100) NOT NULL,
+  `product_desc` text NOT NULL,
+  `product_image` text NOT NULL,
+  `product_unit_price` decimal(6,2) NOT NULL,
+  `admin_id` int(11) NOT NULL,
+  `category_id` int(11) NOT NULL,
+  `total_quantity` int(11) NOT NULL,
+  `status` varchar(100) NOT NULL,
+  PRIMARY KEY (`product_id`),
+  FOREIGN KEY (`category_id`) REFERENCES `categories` (`category_id`),
+  FOREIGN KEY (`admin_id`) REFERENCES `user_admins` (`admin_id`)
+);
+
+-- --------------------------------------------------------
+--
+-- Table structure for table `customers`
+--
+CREATE TABLE `customers` (
+  `customer_id` int(11) NOT NULL AUTO_INCREMENT,
+  `customer_first_name` varchar(100) NOT NULL,
+  `customer_last_name` varchar(100) NOT NULL,
+  `customer_email_address` varchar(100) NOT NULL,
+  `customer_password` varchar(100) NOT NULL,
+  PRIMARY KEY (`customer_id`),
+  FOREIGN KEY (`contact_details_id`) REFERENCES `contact_details` (`contact_details_id`)
+);
+
+-- --------------------------------------------------------
+--
+-- Table structure for table `contact_details`
+--
+CREATE TABLE `contact_details` (
+  `contact_details_id` int(11) NOT NULL AUTO_INCREMENT,
+  `street_address` varchar(255) NOT NULL,
+  `city` varchar(100) NOT NULL,
+  `state` varchar(100) NOT NULL,
+  `zip_code` varchar(100) DEFAULT NULL,
+  `country` varchar(100) NOT NULL,
+  `phone_number` varchar(100) NOT NULL,
+  PRIMARY KEY (`contact_details_id`)
+);
+
+-- --------------------------------------------------------
+--
+-- Table structure for table `orders`
+--
+CREATE TABLE `orders` (
```

Comment:
customer_id should be a column on this table and also a foreign key that references the id on the customers table